lib/parse.js

```diff
@@ -6,14 +6,19 @@ const assign = utils.assign;
 const isEmpty = utils.isEmpty;
 const typedVal = utils.typedVal;
 
-const COLUMN_DELIMITER = ' | ';
+const COLUMN_DELIMITER = '|';
 
 function processPlaintext(plaintext) {
   if (!plaintext) return plaintext;
 
   let lines = plaintext.split('\n');
-  return lines.map(line =>
-    line.split(COLUMN_DELIMITER).map(column => column.trim()));
+
+  // Trim whitespace out of individual lines;
+  // remove empty leftover lines
+  return lines.map(line => line.split(COLUMN_DELIMITER)
+                               .map(column => column.trim())
+                               .filter(column => column))
```

Comment:
Extract this column filtering to separate helper, something like:
```js
function parseLine(line, columnDelimiter=COLUMN_DELIMITER) {
   let columns = line.split(columnDelimiter).map(column => column.trim());
   return columns.filter(column => column.length > 0);
}
```

Then you can use it in following manner:
```js
return lines.map(line => parseLine(line));
```